packages/shopify-app-remix/src/server/authenticate/admin/strategies/__tests__/auth-code-flow/ensure-installed-on-shop.test.ts

```diff
@@ -127,34 +127,9 @@ describe('authorize.admin doc request path', () => {
       );
     });
 
-    it("redirects to the embedded app URL if there is a valid session but the app isn't embedded yet", async () => {
```

Comment:
I moved this test to the `doc-request-path` test ([here](https://github.com/Shopify/shopify-app-js/pull/509/files/bfc0a861cc9301cd7b25edcd3c6790ca96d4045d#diff-3bb6fcab5d11065de680f55d19e94ce514f4b58d39e05e0711b386779ca9c6ce)) because the embedded check always happens, not only in the auth code flow.